politeiawww/backend.go

```diff
@@ -286,6 +315,18 @@ func (b *backend) validateProposal(np www.NewProposal) error {
 		}
 	}
 
+	// proposal title validation
+	name, err := getProposalName(np.Files)
+	if err != nil {
+		return err
+	}
+	if !util.IsValidProposalName(name) {
+		return www.UserError{
+			ErrorCode:    www.ErrorStatusProposalInvalidTitle,
+			ErrorContext: []string{www.ErrorContextProposalInvalidTitle},
```

Comment:
I would rather just pass the regex itself for the error context, so we can keep english text out of it. This will make i18n easier for the client later on.

Reply:
done